docs/style-guide-for-curriculum-challenges.md

```diff
@@ -123,8 +123,19 @@ Here are specific formatting guidelines for challenge text and examples:
 
 - Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
 - The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags
-- Single line code examples go in `<code>` tags
-- Multi-line code examples go in `<blockquote>` tags, and use the `<br>` tag to separate lines. For HTML examples, remember to use escape characters to represent the angle brackets
+- Small pieces of code examples go in `<code>` tags. See example below:
+````
+<code>thisIsAnExampleFunction()</code>
```

Comment:
The following is a typical use of code tags in the instructions section or a test's`text`.
```suggestion
Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
```